app/controllers/messages_controller.rb

```diff
@@ -14,9 +13,9 @@ def index
   def new
     if @group
       @message = @group.messages.new
-    elsif params[:to_person_id] and @person = Person.find(params[:to_person_id])
+    elsif @person = Person.find_by_id(params[:to_person_id])
```

Comment:
You changed the meaning of this code. The previous version would only issue a database query if the `to_person_id` param was present. Thew new line of code always issues a database query, even if it is unnecessary.